package.json

```diff
@@ -6,7 +6,11 @@
   "license": "MIT",
   "homepage": "https://github.com/giakki/uncss",
   "scripts": {
-    "test": "grunt test"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "eslint": "eslint **/*.js",
```

Comment:
Should be

    "eslint": "eslint \"**/*.js\"",

Since not all shells expand the globstar (`**`) correctly. Double-quoting passes the glob to eslint, which will properly expand it.